recipe/meta.yaml

```diff
@@ -1,59 +1,64 @@
-{% set version = "1.3.33" %}
+{% set version = "1.3.35" %}
 
 package:
   name: graphicsmagick
   version: {{ version }}
 
 source:
-  url: https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/{{ version }}/GraphicsMagick-{{ version }}.tar.bz2
-  sha256: d18aaca2d79a10270d49ad1aaa01dce24752f7548880138d59874a78ac62e11f
+  - url: https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/{{ version }}/GraphicsMagick-{{ version }}.tar.bz2
+    sha256: 7fd0b49903eb35df4ab5ca76139ab3f023f47485a4913fbdcf5c642c6e5f01fc
+  - path: CMakeLists.txt
 
 build:
   number: 0
   # ghostscript and gnuplot not available
-  skip: True  # [win]
+  skip: True  # [win and vc<14]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - cmake  # [win]
+    - ninja  # [win]
   host:
-    - ghostscript
+    - ghostscript  # [unix]
+    - gnuplot  # [osx]
+    - libwebp  # [unix]
+    - xorg-libsm  # [osx]
```

Comment:
not sure this is correct for xorg on Linux ... 

maybe adding `libxkbcommon` would also fix the weird unicode issue?
https://anaconda.org/conda-forge/libxkbcommon

Reply:
`libxkbcommon` is already in the dependency graph originally. Another idea is to disable the `Linux` build for `1.3.35` entirely. There is already a `Linux` build for `1.3.33` and the community still can make use of it. And we can tackle `Linux` build later. cc @bgruening

Btw, this is a dep for [`navigation2`](https://ros-planning.github.io/navigation2/) on `ros2`. JFYI.